signature/envelope/cose/cose.go

```diff
@@ -0,0 +1,108 @@
+package cose
```

Comment:
COSE has a draft spec for RFC 3161 ref:  https://www.ietf.org/id/draft-ietf-cose-tsa-tst-header-parameter-01.html

We should get this implemented as part of the first release